### DISCUSSION TITLE
Available on DockerHub?

### DISCUSSION THREAD

**Discussion by litetex** [zitadel/zitadel#8061] [Q&A]
Is the Docker/Containerimage also available somewhere on DockerHub (a quick search yielded no results) or somewhere else besides GitHub?

Asking because [ghcr.io only supports IPv4](https://github.com/orgs/community/discussions/10539) and I would like to deploy the image on an IPv6 only host...

**Comment by fforootd** [comment]
Not yet, we where worried about the download limits of dockerhub.

But we could try and apply for the OSS program and see how it goes 😁

**Reply by litetex** [reply]
Well I don't see any harm in providing the images also there...
You don't have to primarily use DockerHub as distribution platform, but you can use it as a fallback for users like me.
People can still download unlimited times from ghcr if they have an IPv4 :D

And I wouldn't worry about download limits so much since [these limits only apply to the users who pull](https://docs.docker.com/docker-hub/download-rate-limit/).
I personally think that the limits are quite generous (100 pulls per IP per 6h) so if you don't run a quite large buisness that pulls hundreds of images per hour - without a pull-through cache - I think you should be fine.

**Comment by litetex** [comment]
In the meantime [I have worked this around by setting up a personal mirror to DockerHub](https://hub.docker.com/r/litetex/ghcr.zitadel.zitadel).
